sqlite/lib/SQLiteService.js

```diff
@@ -232,6 +232,31 @@ class SQLiteService extends SQLService {
       throw _not_unique(err, 'UNIQUE_CONSTRAINT_VIOLATION') || err
     }
   }
+
+  _convertStreamValues(values) {
+    let any
+    values.forEach((v, i) => {
+      if (v && typeof v.pipe === 'function') {
```

Comment:
Checking for the `pipe` function can be a bit dangerous. As the follow up code is using `on` to the value. Which might not be there. By checking for the `Readable` inheritance it should guarantee most of the expected APIs to be available. 

```suggestion
      if (v && v instanceof Readable) {
```